info.json

```diff
@@ -132129,6 +132129,15 @@
 					"hadith": 53
 				}
 			},
+			{
+				"hadithnumber": 402.2,
+				"arabicnumber": 402.2,
```

Comment:
decimals  in `arabicnumber` field are usually quoted in this api. Anyways I will merge this. I hope this doesn't introduce any breaking changes for others